cypress/support/fragments/inventory/inventorySearchAndFilter.js

```diff
@@ -685,4 +686,15 @@ export default {
   checkSearchQueryText(text) {
     cy.expect(keywordInput.has({ value: text }));
   },
+
+  browseOptionsDropdownIncludesOptions(options) {
+    const browseOptionsDropdown = Select('Search field index');
+    cy.do(browseButton.click());
+    // cypress can't draw selected option without wait
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(ONE_SECOND);
+    options.forEach((name) => {
+      cy.expect(browseOptionsDropdown.find(Option(name)).exists());
```

Comment:
Should work without timeout with 
      cy.expect(browseOptionsDropdown.has({ content: including(name) }));

Reply:
done